src/app/pages/network/explorer/transfer/transfer-list/transfer-list.component.ts

```diff
@@ -302,15 +304,17 @@ export class TransferListComponent extends PaginatedListComponentBase<pst.Event
 
       if (typeof attributes === 'string') {
         for (let name of attrNames) {
-          const match = new RegExp(`"${name}": ?\"?(\\d+)\"?`).exec(attributes);
+          const match = attributes.match(new RegExp(`"${name}": ?"?([+-]?(?:\\d+(?:\\.\\d*)?|\\.\\d+)(?:[eE][+-]?\\d+)?)`));
```

Comment:
Do you know why it's matching w regex instead of using `JSON.parse`?
I feel like that would be more clear.

Reply:
I don't know the original motivation but if we use the JSON.parse and for some reason fails we should add a try/catch block and maybe include the regex matching in the catch as fallback method

Do you want me to change it to JSON?

Reply:
Yes, let's do that